src/gui/addnewtorrentdialog.cpp

```diff
@@ -62,16 +61,14 @@
 
 namespace
 {
-#define SETTINGS_KEY(name) QStringLiteral("AddNewTorrentDialog/" name)
-    const QString KEY_ENABLED = SETTINGS_KEY("Enabled");
-    const QString KEY_DEFAULTCATEGORY = SETTINGS_KEY("DefaultCategory");
-    const QString KEY_TREEHEADERSTATE = SETTINGS_KEY("TreeHeaderState");
-    const QString KEY_WIDTH = SETTINGS_KEY("Width");
-    const QString KEY_EXPANDED = SETTINGS_KEY("Expanded");
-    const QString KEY_TOPLEVEL = SETTINGS_KEY("TopLevel");
-    const QString KEY_SAVEPATHHISTORY = SETTINGS_KEY("SavePathHistory");
-    const QString KEY_SAVEPATHHISTORYLENGTH = SETTINGS_KEY("SavePathHistoryLength");
-    const QString KEY_REMEMBERLASTSAVEPATH = SETTINGS_KEY("RememberLastSavePath");
+#define SETTINGS_KEY(name) "AddNewTorrentDialog/" name
```

Comment:
Why the switch of `QStringLiteral`? Wasn't the same thing achieved by the previous macro?

Reply:
`CachedSettingValue` requires taking a `char *` and IMO these strings should follow the kind in torrentcreatordialog.cpp.